src/elements/content-sidebar/stories/__mocks__/MetadataSidebarRedesignedMocks.ts

```diff
@@ -68,7 +68,19 @@ export const mockGlobalMetadataTemplates = {
 export const mockEmptyMetadataInstances = {
     url: `${apiV2Path}/files/${fileIdWithoutMetadata}/metadata`,
     response: {
-        entries: [],
+        entries: [
```

Comment:
Why is this change needed?

Reply:
Hidden instances should be omitted in MetadataSidebarRedesign. I've added it for extra layer of testing to verify this logic.